app/javascript/controllers/media_tag_controller.js

```diff
@@ -50,15 +50,15 @@ export default class extends Controller {
                  isLocationRestricted: dataset.locationRestricted === "true",
                  fileLabel: dataset.fileLabel || '' }
       })
-    
+
     window.dispatchEvent(new CustomEvent('thumbnails-found', { detail: thumbnails }))
   }
 
   // Open the login window in a new window and then poll to see if the auth credentials are now active.
   logIn(event) {
     const windowReference = window.open(event.params.loginService);
     this.loginStart = Date.now();
-    var checkWindow = setInterval(() => {
+    const checkWindow = setInterval(() => {
```

Comment:
Can we update this to match? https://github.com/sul-dlss/sul-embed/blob/99ac3d3e471f30518e4e28bdd13afc31e7ffa4d1/app/javascript/controllers/file_auth_controller.js#L231

Reply:
Also I am more inclined to let than const. I am worried about a weird edge case that probably won't ever happen.

Reply:
I'm not sure I understand the case where checkWindow would ever be reused for a second value.  I think `const` is better when appropriate as `let` tells the reader that you expect it to be used more than one time and it can be confusing if they can't find that other time.